### ISSUE TITLE
How to create Synonyms with XREFs

### ISSUE THREAD

**Issue by daumkep** [althonos/pronto#70]
Hi again,

I try to convert my scripts to the new v2 version of pronto. I' still struggling with a few things. 

In earlier versions, you created a Synonym e.g. like this:
`Synonym(synonym, syn.scope, None, syn.xref))`

the version v2.0.1 doesn't seem to accepet the xrefs any more. I couldn't figure out from the documentation how to create a Synonym incl. xrefs?

Thanks a lot for your help
Philipp

**Comment by althonos** [comment]
Hi @daumkep,

now you create a synonym for a term directly using [`Term.add_synonym`](https://pronto.readthedocs.io/en/stable/api/pronto.Entity.html#pronto.Entity.add_synonym) method:
```python
term = ontology[...]
term.add_synonym("the description", xrefs=["XREF:001", "XREF:002"])
```

otherwise, you can always add the xrefs later:
```python
term = ontology[...]
synonym = term.add_synonym("the description")
synonym.xrefs = {'XREF:001', 'XREF:002'}
```

EDIT: Actually, it seems I forgot to add a setter for `Synonym.xrefs`, this should be considered a bug, I'll make a bugfix shortly

**Comment by althonos** [comment]
Fixed in 1e134a50fa1cd6e813cabe49e0aa1d1907b6a3b8.
